backend/src/main/resources/db/migration/V0.34__create_tables_for_upload_processing_status.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TYPE verkuendung_import_processing_status AS ENUM ('CREATED', 'PROCESSING', 'SUCCESS', 'ERROR');
+
+CREATE TABLE IF NOT EXISTS verkuendung_import_processes (
+    id UUID NOT NULL PRIMARY KEY,
+    status verkuendung_import_processing_status NOT NULL DEFAULT 'CREATED',
+    created_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    started_at TIMESTAMPTZ,
+    finished_at TIMESTAMPTZ
+);
+
+CREATE TABLE IF NOT EXISTS verkuendung_import_process_detail (
+    id UUID NOT NULL PRIMARY KEY,
+    process_id UUID REFERENCES verkuendung_import_processes(id),
```

Comment:
Since there is no detail without a process, i would set the `process_id` to NOT NULL

Reply:
Not adding the NOT NULL because the way hibernating is doing the cascade with the OneToMany + JoinColumn. It is persisting first the child, then the parent and then setting the foreign key, leading to a not-null-constraint-violation if we set the process_id to NOT NULL